pyfr/plugins/fluidforce.py

```diff
@@ -100,8 +114,16 @@ def __init__(self, intg, cfgsect, suffix):
                 eidxs[etype, fidx].append(eidx)
                 norms[etype, fidx].append(mpn[:, None]*npn)
 
+                # Get the flux points position of the given face and element
+                # indices relative to the moment origin
+                if self._mcomp:
+                    fpts_idx = eles.basis.facefpts[fidx]
+                    rfpt = eles.plocfpts[fpts_idx, eidx] - morigin
+                    rfpts[etype, fidx].append(rfpt)
```

Comment:
Will this all go on one line?

Reply:
This is the one-liner respecting the 79 column limit
```python
                    rfpts[etype, fidx].append(
                        eles.plocfpts[eles.basis.facefpts[fidx], eidx] -
                        morigin)
```

I could chop `morigin` to `mo` to fit it in one line.